src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
The other option which allows things to work by default but won't interfere with the normal flow is that the environment variables only have a default value if unset. So in the pure `/usr` case everything works with the default because there's no environment variable set. But if you set RPP it uses the RPP. And if you want to overlay on top of `/usr` you need to add it to your RPP. This means that it only will effect people who are advanced enough to want to create an overlay/modify the RPP. For those users the standard setup file could be provided in `/usr/share` as a way to load the defaults to be able to extend them. Such that to extend the default environment you can source the explicit default environment and then extend it.